packages/cli/src/utils.js

```diff
@@ -59,7 +59,7 @@ function determineChromePath(options) {
     process.env.CHROME_PATH ||
     PuppeteerManager.getChromiumPath(options) ||
     getChromeInstallationsSafe()[0] ||
-    undefined
+    '/usr/bin/chromium-browser'
```

Comment:
I'm planning to undo this change.

Reply:
I'm leaving this in the draft PR so the maintainers can tell me the idiomatic way to set this environment variable.